content/docs/getting-started.md

```diff
@@ -18,19 +18,19 @@ redirect_from:
   - "docs/environments.html"
 ---
 
-This page is an overview of the React documentation and related resources.
+यह पन्ना रियेक्ट के प्रलेखन और उससे सम्बंधित साधनो  का अवलोकन है | 
 
-**React** is a JavaScript library for building user interfaces. Learn what React is all about on [our homepage](/) or [in the tutorial](/tutorial/tutorial.html).
+**रियेक्ट** एक जावास्क्रिप्ट लाइब्रेरी है जो उपयोगकर्ता के इंटरफेसेस बनाने में काम आती है | सीखे रियेक्ट किस चीज़ के बारे में है हमारे [होम पेज](/) पर या इस [टुटोरिअल] पर (/tutorial/tutorial.html).
```

Comment:
This is `सीखे रियेक्ट किस चीज़ के बारे में है हमारे [होम पेज](/) पर या इस [टुटोरिअल] पर (/tutorial/tutorial.html).` literal translation.
Can you re-phrase the sentence into hindi which will have same meaning something like
`Hamare homepage par ya es tutorial par sikhen react kis cheez ke bare mein hai`